src/main/java/io/strimzi/kafka/quotas/AvailableBytesThrottleFactorPolicy.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.strimzi.kafka.quotas;
+
+import java.util.Collection;
+
+/**
+ * Determines if the number of available bytes on any given volume falls below the configured limit.
+ */
+public class AvailableBytesThrottleFactorPolicy implements ThrottleFactorPolicy {
+    private final long availableBytesLimit;
+
+    /**
+     * Creates and configures the throttle factor supplier
+     *
+     * @param availableBytesLimit the minimum number of bytes below which the throttle should be applied.
+     */
+    public AvailableBytesThrottleFactorPolicy(long availableBytesLimit) {
+        this.availableBytesLimit = availableBytesLimit;
+    }
+
+    @Override
+    public double calculateFactor(Collection<VolumeUsage> observedVolumes) {
+        if (shouldThrottle(observedVolumes)) {
+            return 0.0d;
+        } else {
+            return 1.0d;
+        }
+    }
+
+    private boolean shouldThrottle(Collection<VolumeUsage> observedVolumes) {
+        return observedVolumes.stream().anyMatch(volume -> volume.getAvailableBytes() <= availableBytesLimit);
```

Comment:
We plan to address data validity concerns as part of a separate PR to add the [fallback factor](https://github.com/strimzi/proposals/blob/main/047-cluster-wide-volume-usage-quota-management.md#throttle-factor-fallback). So for now we have added filtering to the VolumeSource to prevent it passing on volumes without reported usage.

resolved with fa5bfa3e846272faeeba42aab294147e059157b4